jwst/tweakreg/tweakreg_step.py

```diff
@@ -200,6 +207,61 @@ def process(self, input):
             else:
                 raise e
 
+        if self.align_to_gaia:
+            try:
+                # Get catalog of GAIA sources for the field
+                #
+                # NOTE:  If desired, the pipeline can write out the reference catalog
+                #        as a separate product with a name based on whatever convention
+                #        is determined by the JWST Cal Working Group
+                if self.output_gaia:
+                    output_name = 'fit_{}_ref.ecsv'.format(self.gaia_catalog.lower())
+                else:
+                    output_name = None
+                ref_cat = amutils.create_astrometric_catalog(self.input_models,
+                                                             self.gaia_catalog,
+                                                             output=output_name)
+
+                # Check that there are enough GAIA sources for a reliable/valid fit
+                num_ref = len(ref_cat)
+                if num_ref < self.minobj:
```

Comment:
I guess this is using the same `minobj` step param that's used to do the existing relative alignment. Is there any reason why the two different processes might want/need to have their own unique param values (e.g. perhaps fewer sources are necessary to get a good absolute fit, than a relative fit)?

Reply:
A separate parameter, min_gaia, was added to allow fields with fewer GAIA sources to be aligned.  The default value of 5 is based on evaluation of the HST fits to GAIA, where we are able to fit >90% of the fields when we have at least 5 GAIA sources in the field.